zerver/migrations/0173_user_message_add_is_private_flag.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.13 on 2018-06-14 13:39
+from __future__ import unicode_literals
+
+import bitfield.models
+from django.db import migrations, transaction
+from django.db.backends.postgresql_psycopg2.schema import DatabaseSchemaEditor
+from django.db.migrations.state import StateApps
+from django.db.models import Q, F
+
+def set_initial_value_of_is_private_flag(
+        apps: StateApps, schema_editor: DatabaseSchemaEditor) -> None:
+    UserMessage = apps.get_model("zerver", "UserMessage")
+    UserProfile = apps.get_model("zerver", "UserProfile")
+    user_profiles = UserProfile.objects.all()
```

Comment:
Code looks good overall, but one comment.

It's a simple optimization here to just userids with `.values('id')`.  I know it's not the bottleneck here, but you're not buying much simplicity to pull in the fat user objects for all Nk users.